server/models/Profile.js

```diff
@@ -0,0 +1,45 @@
+const mongoose = require('mongoose');
+
+const profileSchema = new mongoose.Schema({
+    firstName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    lastName: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    description: String,
+    gender: {
+        type: String,
+        required: true,
+        enum: ['Male', 'Female']
+    },
+    birthDate: Date,
+    email: {
+        type: String,
+        lowercase: true,
+        required: true,
+        trim: true,
+        unique: true
+    },
+    phoneNumber: {
+        type: Number,
+        trim: true
+    },
+    address: {
+        type: String,
+        required: true
+    },
+    availability: [
```

Comment:
I think the availability should be in a separated data structure,
my suggestion for ​data structure describing the availability:
`{
    "Start" : HH:MM,
    "End" : HH:MM,
    "date" : "YYYY-MM-DD",
    "user_id" : ....,
    "_id" : ObjectId("xxxxxxxxxxxx")
}`